Algolia.Search/Index.cs

```diff
@@ -1680,6 +1680,12 @@ public Task<JObject> SaveRuleAsync(JObject queryRule, RequestOptions requestOpti
 			}
 
 			string objectID = (string)queryRule["objectID"];
+
+			if (objectID.IsNullOrEmpty)

```

Comment:
woops my bad, that's supposed to be a method, so just add `()` at the end of `IsNullOrEmpty`

That's why the build failed in CI

Reply:
Sorry but the Visual Studio for Mac wasn't helpful on that. It's gonna be easier with a Windows box soon :)